src/main/java/seedu/address/logic/commands/listcommand/ListCommand.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands.listcommand;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+
+/*
+ * This Command is for listing everything (issue, mentor, participant, team).
+ */
+public class ListCommand extends Command {
+
+    /* Possible Fields */
+
+    public static final String COMMAND_TYPE = "list";
+
+    @Override
+    public CommandResult execute(EntityList entityList) throws CommandException {
+        requireNonNull(entityList);
+
+        // Possible format
+        // TeamList teamList = entityList.getTeamList();
+
+        // I think Team should have a method that lists its connections (e.g. Team#showConnection)
```

Comment:
Possible, but it'll be specific to the Team Object and not entity, which means that you might have to downcast the objects before running this.

Currently, the Team Object will expose a getter and setter for the participants and mentors fields. Would this be sufficient? I can add more methods if necessary.